mmv1/third_party/terraform/go.mod.erb

```diff
@@ -0,0 +1,33 @@
+<% autogen_exception -%>
+module github.com/hashicorp/terraform-provider-google<%= "-" + version unless version == 'ga' -%>
+
+require (
+    cloud.google.com/go/bigtable v1.10.1
+    github.com/GoogleCloudPlatform/declarative-resource-client-library v0.0.0-20210803014808-0fdb1dc411b3
+    github.com/apparentlymart/go-cidr v1.1.0
+    github.com/client9/misspell v0.3.4
+    github.com/davecgh/go-spew v1.1.1
+    github.com/dnaeon/go-vcr v1.0.1
+    github.com/gammazero/deque v0.0.0-20180920172122-f6adf94963e4 // indirect
+    github.com/gammazero/workerpool v0.0.0-20181230203049-86a96b5d5d92
+    github.com/golangci/golangci-lint v1.40.1
+    github.com/hashicorp/errwrap v1.0.0
+    github.com/hashicorp/go-cleanhttp v0.5.2
+    github.com/hashicorp/go-multierror v1.1.1
+    github.com/hashicorp/go-version v1.3.0
+    github.com/hashicorp/hcl/v2 v2.6.0 // indirect
+    github.com/hashicorp/terraform-plugin-sdk/v2 v2.5.0
+    github.com/mitchellh/go-homedir v1.1.0
+    github.com/mitchellh/hashstructure v1.1.0
+    github.com/spf13/afero v1.2.2 // indirect
+    github.com/zclconf/go-cty v1.5.1 // indirect
+    golang.org/x/net v0.0.0-20210503060351-7fd8e65b6420
+    golang.org/x/oauth2 v0.0.0-20210514164344-f6687ab2804c
+    golang.org/x/sys v0.0.0-20210611083646-a4fc73990273 // indirect
+    golang.org/x/tools v0.1.3 // indirect
+    google.golang.org/api v0.48.0
+    google.golang.org/grpc v1.38.0
+)
+    
+go 1.16
+// test if this file is copied
```

Comment:
```suggestion
```

Reply:
Yep!  You just add the PR number to the known-safe list on https://console.cloud.google.com/functions/details/us-central1/disallow-evil-builds?project=graphite-docker-images (whose name I can safely post here since it's IAM access controlled!)

I've done it just now for this PR.  Rerunning should work.

Reply:
Thanks!